### PR TITLE
Fix(refs:T36703): unused checkbox

### PR DESCRIPTION
**Pull request by elmasdemos**
**Ticket:** https://yaits.demos-deutschland.de/T36703

### Description:

- the checkbox cannot be used in `Mitgezeichnete Stellungnahmen`
- the externId is missing and the hash should be besides the number/externId 
- `Alle markieren` should be shown in one line, if the display has enough size

## Ihre eingereichten Stellungnahmen
screenshot_1

## Mitgezeichnete Stellungnahmen
screenshot_2

### PR Checklist

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/components/statement/publicStatementLists/DpPublicStatement.vue [hunk 1]
```diff
@@ -23,15 +23,18 @@
...
-              data-cy="statementNumber"
-              class="inline u-mb-0 u-ml-0_25">{{ number || '' }}</label>
+            <span
+              class="c-public-statement__tooltip"
+              v-tooltip="renderTooltipContent(tooltipContent)">
+              <label
+                :for="number"
+                data-cy="statementNumber"
+                class="inline u-mb-0">
+                {{ `#${ number || externId }` }}
```
Comment (spiess-demos) [inline client/js/components/statement/publicStatementLists/DpPublicStatement.vue:33]: I'd suggest to remove the `#` hash character here. We could as well use an info icon as the tooltip trigger (positioned after the ID).

Another question: isn't "number" and "externId" the same value?

File: templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig [hunk 2]
```diff
@@ -40,7 +40,7 @@
     small: ' u-1-of-4',
     smaller: ' u-1-of-5',
     tiny: ' u-1-of-6',
-    mini: ' u-1-of-8',
+    mini: ' u-1-of-2',
```
Comment (spiess-demos) [inline templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig:43]: Why do you apply this change? I do not see the need to do so within this PR, also, i guess it changes layout in many places where `element.html.twig` is used.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/statement/publicStatementLists/DpPublicStatement.vue (hunk 1 above)
Comment (elmasdemos) [reply client/js/components/statement/publicStatementLists/DpPublicStatement.vue:33]: you have a suggestion, which icon i could use here? I would suggest just to remove the hash.

No, they are different, but its difficult to explain it here

File: templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig (hunk 2 above)
Comment (elmasdemos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig:43]: the `Alles markieren` looks like this way 
markerall

File: client/js/components/statement/publicStatementLists/DpPublicStatement.vue (hunk 1 above)
Comment (elmasdemos) [reply client/js/components/statement/publicStatementLists/DpPublicStatement.vue]: I decided to remove it. done here https://github.com/demos-europe/demosplan-core/pull/2832/commits/da48392fa44c9bf01263c06810ee84d435a39fc3

File: templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig (hunk 2 above)
Comment (elmasdemos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig]: done https://github.com/demos-europe/demosplan-core/pull/2832/commits/bfe56a2aee460e5ba39af47763135addad439511

File: client/js/components/statement/publicStatementLists/DpPublicStatement.vue [hunk 3]
```diff
@@ -240,7 +243,7 @@
 </template>
 
 <script>
-import { CleanHtml, DpFlyout, DpInlineNotification } from '@demos-europe/demosplan-ui'
+import { CleanHtml, DpFlyout, DpInlineNotification, DpContextualHelp } from '@demos-europe/demosplan-ui'
```
Comment (spiess-demos) [inline client/js/components/statement/publicStatementLists/DpPublicStatement.vue:246]: can be removed if not used.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig (hunk 2 above)
Comment (elmasdemos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig]: yes did it here https://github.com/demos-europe/demosplan-core/pull/2832/commits/bfe56a2aee460e5ba39af47763135addad439511

File: client/js/components/statement/publicStatementLists/DpPublicStatement.vue [hunk 4]
```diff
@@ -249,6 +252,7 @@ export default {
   name: 'DpPublicStatement',
 
   components: {
+    DpContextualHelp,
```
[the remark this answers is not in the document]
Comment (elmasdemos) [reply client/js/components/statement/publicStatementLists/DpPublicStatement.vue]: thanks done https://github.com/demos-europe/demosplan-core/pull/2832/commits/0e05eed35480ba1137ca30bb5d4eea0ca477e2ab

File: client/js/components/statement/publicStatementLists/DpPublicStatement.vue (hunk 3 above)
Comment (elmasdemos) [reply client/js/components/statement/publicStatementLists/DpPublicStatement.vue]: thanks done https://github.com/demos-europe/demosplan-core/pull/2832/commits/0e05eed35480ba1137ca30bb5d4eea0ca477e2ab

File: templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig (hunk 2 above)
Comment (spiess-demos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig]: i guess the `0` should not be a string but an actual number.

Comment (elmasdemos) [reply templates/bundles/DemosPlanCoreBundle/UI/form/element.html.twig]: I see <.< I was wondering, why it ditnt look like I expect it. Now it looks fine, thanks. done here https://github.com/demos-europe/demosplan-core/pull/2832/commits/10adbfac7a6ec2e91a3f0c7ef65f2f57277fed32

**Review by spiess-demos — APPROVED** [review]
lgtm (code review only)

**Pull request merged**